components/mpas-ocean/src/Registry.xml

```diff
@@ -734,14 +737,39 @@
 					possible_values=".true. or .false."
 		/>
 		<nml_option name="config_sgr_temperature_prescribed" type="real" default_value="0.0" units="C"
-			        description="Prescribed subglacial runoff temperature value, applied when config_use_sgr_opt_temp_prescribed = .true."
+			                description="Prescribed subglacial runoff temperature value, applied when config_use_sgr_opt_temp_prescribed = .true."
 					possible_values="Any real number."
 		/>
 		<nml_option name="config_sgr_salinity_prescribed" type="real" default_value="0.0" units="1.e-3"
 					description="Prescribed subglacial runoff salinity value, applied when config_use_sgr_opt_salt_prescribed = .true."
 					possible_values="Any positive real number"
+                />
+        </nml_record>
+	<nml_record name="precip_scaling" mode="init;forward">
+                <nml_option name="config_use_precip_scaling" type="logical" default_value=".false."
+			                description="If enabled, precipitation fluxes from the data atmosphere are scalled to ensure freshwater balance in the ocean model."
+					possible_values=".true. or .false."
+                />
+	        <nml_option name="config_precip_scaling_mode" type="character" default_value="constant"
+			                description="Option for precipitation scaling to ensure freshwater balance in the ocean model."
+                        		possible_values="'constant' or 'time-dependent'"
```

Comment:
Here and elsewhere, indentation in Registry.xml is with tabs, not spaces.  It's annoying and others have not always done this correctly but let's not add more inconsistency..